### DISCUSSION TITLE
Newbie question. Access data from selected rows

### DISCUSSION THREAD

**Discussion by renrom** [Budibase/budibase#9314] [Help] [answered]
I have a form with a table with allow row selection. 

I want to select a couple of rows and from the selected rows i would want to fill one field on the form with a column from the selected fields. Is that possible?

I tried the {{ table.Selected rows }} but then i get only the ID if i am right.

**Comment by renrom** [comment, marked as answer]
Never mind. i did find a working solution within this https://github.com/Budibase/budibase/discussions/3034

**Reply by melohagan** [reply]
This may help as well: https://github.com/Budibase/budibase/discussions/8759#discussioncomment-4207094
